### DISCUSSION TITLE
About page reloading, does Rspress supports md and mdx reload after saving

### DISCUSSION THREAD

**Discussion by Jess-Qiu** [web-infra-dev/rspress#797] [Q&A] [answered]
你好，这边在编辑 md、mdx 文件保存后未执行 reload，倒是保存 _meta.json 文件会执行 reload 操作
![image](https://github.com/web-infra-dev/rspress/assets/22249167/5d3a7139-8367-4b5f-a43b-e627a6f1d4dc)

**Comment by sanyuan0704** [comment, marked as answer]
编辑 md(x) 一般会进行热更新操作哈，目前的版本 (1.15.1) 如果是更新标题和侧边栏会自动进行 page reload。
